DALOptimizer/PrintFunction.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Linq;
+
+using ICSharpCode.NRefactory.CSharp;
+
+namespace DALOptimizer
+{
+    class PrintFunction
+    {
+        public void PrintMethod(Solution solution)
+        {
+            Console.WriteLine("Found {0} places to refactor Field Declaration in {1} files.",
+                  solution.AllFiles.Sum(f => f.IndexOfFieldDecl.Count),
+                  solution.AllFiles.Count(f => f.IndexOfFieldDecl.Count > 0));
```

Comment:
@sushantyelpale `f.IndexOfFieldDecl.Count` is performed twice, try to perform it only once if it recalculates it every time we call it.